PicPi/FrameAPI.swift

```diff
@@ -12,6 +12,12 @@ enum FrameAPIErrors: Error {
   case dataMalformed
 }
 
+struct PhotoData: Decodable {
```

Comment:
wdyt of changing it to photoInfo?

Reply:
sgtm wha about PhotosInfoData? the data part is so we know this is a model from json aka the HTTP API